libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -142,24 +142,28 @@ def _convert_to_parts(message: BaseMessage) -> List[Part]:
         return [_convert_to_prompt(part) for part in raw_content]
 
     vertex_messages = []
-    raw_system_message = None
+    convert_system_message_to_human_content = None
+    system_instructions = None
     for i, message in enumerate(history):
         if (
             i == 0
             and isinstance(message, SystemMessage)
             and not convert_system_message_to_human
```

Comment:
we can probably drop convert_system_message_to_human now, wdyt?

Reply:
I thought keeping it for backwards compatibility